### PR TITLE
06.01 List news at the home page

### PR DESCRIPTION
**Pull request by yevhverb**
- created articles list widget;
- changed home page container;

image

### CODE REVIEW

File: right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const ArticlesListWidget = ({ data }) => {
+  return (
+    <article className="h-full flex flex-col justify-between rounded-xl text-lightgray-700">
+      { data.slice(0, 3).map(item => <ArticlesListWidgetItem key={item.id} data={item} />) }
+      <div className="w-full flex justify-center">
+        <Link 
+          to="/news"
```
Comment (alex-boklag) [inline right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx]: I think that Link should lead to the HappyStories page.

File: right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          className="p-2 px-5 bg-white shadow-xl rounded-xl hover:bg-green-300 hover:text-green-700">
+          Больше новостей
+        </Link>
+      </div>
+    </article>
+  );
+};
+
+const ArticlesListWidgetItem = ({ data: { id, title, photo } }) => (
+  <Link to={`/news/${id}`}>
```
Comment (alex-boklag) [inline right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx]: I think that Link should lead to the HappyStory page.

File: right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx (hunk 1 above)
Comment (yevhverb) [reply right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx]: Сorrected!

File: right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx (hunk 2 above)
Comment (yevhverb) [reply right-for-life_front/src/components/ArticlesListWidget/ArticlesListWidget.jsx]: Сorrected!

**Pull request merged**